openstack/compute/v2/servers/results.go

```diff
@@ -212,6 +212,9 @@ type Server struct {
 	// to it.
 	SecurityGroups []map[string]interface{} `json:"security_groups"`
 
+	// AttachedVolumes includes the volume attachments of this instance
+	AttachedVolumes []map[string]string `json:"os-extended-volumes:volumes_attached"`
```

Comment:
Instead of `[]map[string]string`, how about:

```go
type AttachedVolume struct {
  ID string `json:"id"`
}
```

and then

```go
// AttachedVolumes includes the volume attachments of this instance
AttachedVolumes []AttachedVolume `json:"os-extended-volumes:volumes_attached"`
```

According to the code, `id` is the only field except if microversion 2.3+ is used, and then there is a `delete_on_termination` field. But since that's a microservice addition, we'll have to handle that differently, so let's not worry about that in this PR.